internal/sbom/mod_1.18.go

```diff
@@ -32,3 +33,29 @@ func ParseBuildInfo(data string) (*BuildInfo, error) {
 	bi := BuildInfo(*dbi)
 	return &bi, nil
 }
+
+func modulePackageName(mod *debug.Module) string {
```

Comment:
Should these be in some shared file that both 1.18 and pre-1.18 code can reuse?

Reply:
Boooo I guess what you had was fine. I hate this pre-1.18 and count down the days until we don't have to support it.

Reply:
Yeah, I had no idea until CI blew up on me, and I had to contort things to be able to leverage module types.

Will be good to kill this with 🔥